components/InfoSectionTaskForce/index.tsx

```diff
@@ -22,13 +23,19 @@ const roleDisplayName = {
   member: "Medlem",
 };
 
-const InfoSectionStudy = ({ taskforce }: { taskforce: Taskforce | null }) => {
+const InfoSectionTaskforce = ({
+  taskforce,
+}: {
+  taskforce: Taskforce | null;
+}) => {
   if (!taskforce) return null;
 
   return (
     <InfoSectionWrapper id="taskforce">
       <h2 className={styles.title}>1. Klasse Taskforce</h2>
-      <p className={styles.description}>{taskforce.description}</p>
+      <div className={styles.textWrapper}>
+        {taskforce.description && <PortableText value={taskforce.description} />}
+      </div>
```

Comment:
Is it intentional that you still render the wrapper even when there is no description?

Reply:
Yes, `className` is not a property of `PortableText` so I just let it be like that from the previous solution. Could probably avoid it and apply css differently, I just didnt bother 😅

Reply:
Ah fair enough. Won't make that much of a difference seeing that there should always be a description, but if there isn't one there will be some unnecessary margin above and beneath the empty container.

So what I was wondering was why you did what you did and not 
```suggestion
        {taskforce.description && 
        	(<div className={styles.textWrapper}>
        		<PortableText value={taskforce.description} />
      		</div>)
      	} 
```

Although I see that the behaviour in your edit is the way it was done previously, so it won't be much of a change either way I guess